lib/home_screen.dart

```diff
@@ -95,7 +95,7 @@ class _HomeScreenState extends State<HomeScreen> {
                     ElevatedButton.icon(
                       style: ElevatedButton.styleFrom(
                         padding: const EdgeInsets.symmetric(
-                            horizontal: 32, vertical: 24),
+                            horizontal: 16, vertical: 12),
```

Comment:
I needed to reduce the padding for my screen size. I'm not sure, but are there media-queries or relative sizes which we could use depending on the screen dimensions?